### PR TITLE
dx-1197 Added MPO Capture

### PR DESCRIPTION
**Pull request by arebra**
Added MPO capture section

### CODE REVIEW

File: payments/mobile-pay/capture.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+title: MobilePay Payments – Capture
```
Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737&bid=20167196)

File: _includes/mobile-pay-payments-capture.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+## Capture
```
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: payments/mobile-pay/capture.md (hunk 1 above)
Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/mobile-pay-payments-capture.md (hunk 2 above)
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

**Comment by arebra** [comment]
I need to add a more generic if in the transaction-response.md and transaction-list-response.md. I know 👍

File: _includes/mobile-pay-payments-capture.md (hunk 2 above)
Comment (asbjornu) [reply _includes/mobile-pay-payments-capture.md:1]: Add a newline at the end of `_includes/mobile-pay-payments-capture.md` please.

**Review by asbjornu — CHANGES_REQUESTED** [review]

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737&bid=20167196)

File: payments/mobile-pay/capture.md (hunk 1 above)
Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/mobile-pay-payments-capture.md (hunk 2 above)
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: payments/mobile-pay/capture.md (hunk 1 above)
Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/mobile-pay-payments-capture.md (hunk 2 above)
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/payment-transaction-states.md [hunk 3]
```diff
@@ -14,9 +14,9 @@ the course of a purchase. You can find a short description of each state below.
     Examples of possible failures are triggering of anti-fraud protection or if
     the payer reaches the maximum number of attempts for a given payment.
 
-*   `aborted` means that the merchant has aborted the payment before the end user
-    has fulfilled the payment process. This is done by calling the `PATCH` function
-    for abort operations.
+*   `aborted` means that the merchant has aborted the payment before the end
+    user has fulfilled the payment process. This is done by calling the `PATCH`
+    function for abort operations.
```
Comment (asbjornu) [inline _includes/payment-transaction-states.md:19]:
```suggestion
*   `aborted` means that the merchant has aborted the payment before the
    payer has fulfilled the payment process. Aborting a payment is done by
    performing the `update-payment-abort` operation.
```

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: _includes/payment-transaction-states.md (hunk 3 above)
Comment (arebra) [reply _includes/payment-transaction-states.md:19]: That include wasn't supposed to be a part of this commit, but a great suggestion nonetheless! 👍

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737&bid=20167196)

File: payments/mobile-pay/capture.md (hunk 1 above)
Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/mobile-pay-payments-capture.md (hunk 2 above)
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: payments/mobile-pay/capture.md (hunk 1 above)
Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/mobile-pay-payments-capture.md (hunk 2 above)
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/delete-token.md [hunk 4]
```diff
@@ -1,45 +0,0 @@
-{% assign documentation_section = include.documentation_section %}
```
Comment (asbjornu) [inline _includes/delete-token.md:1]: Should `delete-token.md` be deleted?

File: _includes/mobile-pay-payments-capture.md (hunk 2 above)
Comment (asbjornu) [reply _includes/mobile-pay-payments-capture.md:1]: Can you add a newline to the end of `_includes/mobile-pay-payments-capture.md `?

**Review by asbjornu — DISMISSED** [review]

File: _includes/delete-token.md (hunk 4 above)
Comment (arebra) [reply _includes/delete-token.md:1]: No! I deleted it by mistake. I commited dx-1171 earlier today where I changed something in it again, so I hope I can fix it in a merge conflict?

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737&bid=20167196)

File: _includes/mobile-pay-payments-capture.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+sequenceDiagram
+    participant SwedbankPay as Swedbank Pay
+
+    Merchant->>SwedbankPay: POST <mobilepay capture>
+    activate Merchant
+    activate SwedbankPay
+    SwedbankPay-->>Merchant: transaction resource
+    deactivate SwedbankPay
+    deactivate Merchant
+```
```
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:62]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/mobile-pay-payments-capture.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
+## Capture
```
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: payments/mobile-pay/capture.md (hunk 1 above)
Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/delete-token.md (hunk 4 above)
Comment (asbjornu) [reply _includes/delete-token.md:1]: You can just check it out from `develop` again by doing the following on the command line:

```bash
git checkout develop -- _includes/delete-token.md
```

That will bring the file back, so you can commit it again. The ideal would be to change the commit where you removed it so it isn't removed, but I think we need to do that on a call.

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737&bid=20167196)

File: payments/mobile-pay/capture.md (hunk 1 above)
Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/mobile-pay-payments-capture.md (hunk 2 above)
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: payments/mobile-pay/capture.md (hunk 1 above)
Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/mobile-pay-payments-capture.md (hunk 2 above)
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737&bid=20167196)

File: payments/mobile-pay/capture.md (hunk 1 above)
Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/mobile-pay-payments-capture.md (hunk 5 above)
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:62]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/mobile-pay-payments-capture.md (hunk 6 above)
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737&bid=20167196)

File: payments/mobile-pay/capture.md (hunk 1 above)
Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/mobile-pay-payments-capture.md (hunk 5 above)
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:62]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/mobile-pay-payments-capture.md (hunk 6 above)
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 5
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737&bid=20167196)

File: _includes/mobile-pay-payments-capture.md (hunk 2 above)
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: payments/mobile-pay/capture.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
```
Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/mobile-pay-payments-capture.md (hunk 2 above)
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: payments/mobile-pay/capture.md (hunk 7 above)
Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: payments/mobile-pay/capture.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+{% include mobile-pay-payments-capture.md %}
+
+**Et voilà!** The payment should now be complete, secure and
+everyone should be happy. But, sometimes you also need to implement the
+cancellation and reversal operations described in After Payment.
+
+{% include iterator.html prev_href="redirect"
+                         prev_title="Back: Redirect"
+                         next_href="after-payment"
+                         next_title="Next: After Payment" %}
```
Comment (payex-dev) [inline payments/mobile-pay/capture.md:17]: Codacy found an issue: [[no-consecutive-blank-lines] Remove 1 line after node](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
- Added 3
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737&bid=20167196)

File: payments/mobile-pay/capture.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
+title: MobilePay Payments – Capture
```
Comment (payex-dev) [inline payments/mobile-pay/capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

File: _includes/mobile-pay-payments-capture.md (hunk 2 above)
Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [[final-newline] Missing newline character at end of file](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

Comment (payex-dev) [inline _includes/mobile-pay-payments-capture.md:1]: Codacy found an issue: [Command failed: git rev-parse --show-cdup
/bin/sh: git: not found
](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=6178737)

**Review by asbjornu — APPROVED** [review]

**Pull request merged**